### PR TITLE
Color edges instead of static groups

### PR DESCRIPTION
**Pull request by XPerianer**
grafik

### CODE REVIEW

**Comment by robertkovax** [comment]
this is cool! I'm wondering if it should be red or some other signal color?

Rob
---
www.robertkovax.com

File: src/configuration/configuration.rb [hunk 1]
```diff
@@ -90,6 +90,8 @@ module Configuration
   # These are the same colors, but with saturation lower, and a little bit shuffled, to prevent collisions
   UNINTENSE_COLORS = %w[#dcdcdc #696969 #202020 #808080 #c0c0c0].freeze
 
+  DARK_COLOR = "#999999"
```
Comment (birneamstiel) [inline src/configuration/configuration.rb:93]:
```suggestion
  DARK_COLOR = '#999999'.freeze
```
if we want to listen to rubicon ;-)

File: src/export/node_export_visualization.rb [hunk 2]
```diff
@@ -40,10 +40,28 @@ def perform(export_interface)
...
+        edges
+      end
+
+      edge_groups.each do |group|
+        group.each do |edge|
+          edge.link.material = Sketchup.active_model.materials['standard_material']
+        end
+      end
+
+      # No, only the intersections are colored differently (black)
```
Comment (birneamstiel) [inline src/export/node_export_visualization.rb:60]: I guess you meant "now".
Also: maybe you could add that the intersections are the rotation axis?

**Comment by robertkovax** [comment]
eventually the hinging hubs could be also included in the coloring.

Rob
---
www.robertkovax.com

**Pull request merged**